CONTRIBUTING.md

```diff
@@ -0,0 +1,81 @@
+# Contributing to ESPResSo
+[contributing-to-rust]: #contributing-to-rust
```

Comment:
Rust?

Reply:
I copy-pasted the structure from the Rust contributors guide.